projects/breach-notice-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/Contact.kt

```diff
@@ -71,6 +71,10 @@ class Contact(
     @Id
     @Column(name = "contact_id")
     val id: Long,
+
+    @OneToMany
+    @JoinColumn("contact_id")
+    val unpaidWorkAppointments: List<UpwAppointment>? = emptyList(),
```

Comment:
Should this be a `mappedBy`, as the join column is on the other entity?
```suggestion
    @OneToMany(mappedBy = "contact")
    val unpaidWorkAppointments: List<UpwAppointment> = emptyList(),
```

Reply:
done